autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/AutoscaleClusterState.java

```diff
@@ -19,6 +23,14 @@ public void setEnableAutoscaling(boolean enableAutoscaling) {
         this.enableAutoscaling = enableAutoscaling;
     }
 
+    public boolean isUseStopStartMechanism() {
```

Comment:
Nit: Rename to getUseStopStartMechanism (is* made more sense on the name in the last patch)

Reply:
Updated method name